.circleci/config.yml

```diff
@@ -418,6 +439,15 @@ jobs:
           command: swift build -c release --target RevenueCatUI
           no_output_timeout: 30m
 
+  pod-lib-lint:
```

Comment:
new job, just does pod lib lint, but that in turn does a bunch of checks for compilation in different platforms and OS versions, so I've found it to be super useful in the past at catching missing availability checks